edx_rest_framework_extensions/tests/test_jwt_decoder.py

```diff
@@ -188,3 +199,17 @@ def test_decode_unsupported_jwt_version(self):
 
             msg = "All combinations of JWT issuers and secret keys failed to validate the token."
             patched_log.error.assert_any_call(msg)
+
+    def test_upgrade(self):
+        """
+        Verifies the JWT is upgraded when an old (starting) version is provided.
+
+        Compares the decoded token with the latest payload version that would
+        other be generated, while keeping time-related values constant.
+        """
+        jwt_payload = generate_starting_version(self.user)
+        token = generate_jwt_token(jwt_payload)
+
+        upgraded_payload = generate_jwt_payload(self.user, version='1.0.0')
+        upgraded_payload['iat'], upgraded_payload['exp'] = jwt_payload['iat'], jwt_payload['exp']
```

Comment:
Are these the time-related values?  I didn't understand this line at all.  Can we remove the comment from the docstring and instead have this line commented with something like:
```
Keep time-related values constant for easier comparison.
```

Reply:
done.